app/src/main/res/layout/pin_password_activity.xml

```diff
@@ -14,127 +14,186 @@
-      app:itemWidth="28dp"
-      app:layout_constraintEnd_toEndOf="parent"
-      app:layout_constraintStart_toStartOf="parent"
-      app:layout_constraintTop_toBottomOf="@+id/pin_password_error_text_view"
-      app:lineColor="@color/avatarBorder"
-      app:lineWidth="1dp"
-      app:viewType="rectangle" />
-
-    <TextView
-      android:id="@+id/forgot_pin"
-      android:layout_width="wrap_content"
-      android:layout_height="wrap_content"
-      android:fontFamily="sans-serif"
-      android:minHeight="48dp"
-      android:paddingTop="12dp"
-      android:text="@string/pin_password_forgot_pin"
-      android:textAllCaps="true"
-      android:textColor="@color/colorPrimary"
-      android:textSize="14sp"
+    <FrameLayout
+      android:layout_width="0dp"
+      android:layout_height="0dp"
+      android:background="@color/oppiaLightGreen"
+      app:layout_constraintBottom_toBottomOf="parent"
       app:layout_constraintEnd_toEndOf="parent"
       app:layout_constraintStart_toStartOf="parent"
-      app:layout_constraintTop_toBottomOf="@+id/show_pin" />
+      app:layout_constraintTop_toBottomOf="@id/pin_passord_app_bar_layout">
 
-    <LinearLayout
-      android:id="@+id/show_pin"
-      android:layout_width="wrap_content"
-      android:layout_height="wrap_content"
-      android:layout_marginStart="12dp"
-      android:minWidth="48dp"
-      android:minHeight="48dp"
-      android:orientation="vertical"
-      app:layout_constraintBottom_toBottomOf="@+id/input_pin"
-      app:layout_constraintStart_toEndOf="@+id/input_pin"
-      app:layout_constraintTop_toTopOf="@+id/input_pin">
-
-      <ImageView
-        android:id="@+id/show_hide_password_image_view"
-        android:layout_width="32dp"
-        android:layout_height="32dp"
-        android:layout_gravity="center_horizontal"
-        android:contentDescription="@{viewModel.showPassword ? @string/password_shown_icon : @string/password_hidden_icon}"
-        android:src="@{viewModel.showPassword ? @drawable/ic_show_eye_icon : @drawable/ic_hide_eye_icon}" />
-
-      <TextView
-        android:id="@+id/show_hide_password_text_view"
+      <androidx.constraintlayout.widget.ConstraintLayout
         android:layout_width="wrap_content"
         android:layout_height="match_parent"
-        android:layout_gravity="center_horizontal"
-        android:layout_marginTop="4dp"
-        android:fontFamily="sans-serif"
-        android:text="@{viewModel.showPassword ? @string/pin_password_hide : @string/pin_password_show}"
-        android:textAllCaps="true"
-        android:textColor="@color/colorPrimary"
-        android:textSize="12sp" />
-    </LinearLayout>
+        android:layout_gravity="center"
+        android:paddingTop="42dp">
+
+        <TextView
+          android:id="@+id/hello_text_view"
+          android:layout_width="wrap_content"
+          android:layout_height="wrap_content"
+          android:fontFamily="sans-serif-medium"
+          android:lines="1"
```

Comment:
Why does it need to be in `1` line only? I think it should be completely fine to expand it to mutliple lines as we do have space on this screen.

Reply:
removed this line.